qemu/tests/cfg/test_vdpa_control_virtqueue.cfg

```diff
@@ -0,0 +1,14 @@
+- test_control_virtqueue:
+    type = test_vdpa_control_virtqueue
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    only Linux
+    nettype_fixed = vdpa
```

Comment:
As when `nettype = bridge` we can set `only bridge`, so I would suggest also adding `only vdpa`, so other nettype will not test this case.

Reply:
Done.